lib/mail_catcher/mail.rb

```diff
@@ -157,4 +157,13 @@ def delete_message!(message_id)
     @delete_messages_query.execute(message_id) and
     @delete_message_parts_query.execute(message_id)
   end
+
+  def delete_older_messages!(max_mail_to_keep = MailCatcher::options[:max_mail_to_keep])
+    return if max_mail_to_keep <= 0
+    @delete_older_messages_query ||= db.prepare "DELETE FROM message WHERE id NOT IN (SELECT id FROM message ORDER BY created_at DESC LIMIT ?)"
```

Comment:
Since `id` is the primary key, it has an index. Can we `ORDER BY id`?
Also, dealing with just a single indexed column, most databases run much quicker, avoiding hitting the data page altogether.
It may be just a marginal savings though. So feel free to ignore